oneview_redfish_toolkit/api/scmb.py

```diff
@@ -110,19 +110,37 @@ def get_oneview_client():
         # multiple OneViews for events service
         ip=config.get_oneview_multiple_ips()[0],
         credentials=config.get_credentials(),
-        api_version=300
+        api_version=600
     )
     ov_client = OneViewClient(ov_config)
     ov_client.connection.login(config.get_credentials())
 
     return ov_client
 
 
+def _get_ca_cert(ov_client):
+    con = ov_client.connection
+    URI = '/rest/certificates/ca'
+    resource_client = ResourceClient(con, URI)
+    cert = resource_client.get(URI + "?filter=certType:INTERNAL")
+    if isinstance(cert, dict):
+        returnCert = None
+        if 'members' in cert.keys():
+            for certObj in cert.get('members'):
+                if certObj and certObj.get('certificateDetails') and \
+                        certObj.get('certificateDetails').get('base64Data'):
+                    returnCert = certObj.get('certificateDetails').\
+                        get('base64Data')
+                    break
+        return returnCert
+    # If cert is not a dictionary then returning None
+    return None
```

Comment:
Maybe you could improve the code:
    returnCert = None
    if isinstance(cert, dict):
		if 'members' in cert.keys():
            for certObj in cert.get('members'):
                if certObj and certObj.get('certificateDetails') and \
                        certObj.get('certificateDetails').get('base64Data'):
                    returnCert = certObj.get('certificateDetails').\
                        get('base64Data')
                    break
    return returnCert

Reply:
Sure will do it